src/e3/net/http.py

```diff
@@ -74,7 +74,7 @@ class HTTPSession:
     CHUNK_SIZE = 1024 * 1024
     DEFAULT_TIMEOUT = (60, 60)
 
-    def __init__(self, base_urls: Optional[Union[List[str], List[BaseURL]]] = None):
+    def __init__(self, base_urls: Optional[List[Union[str, BaseURL]]] = None):
```

Comment:
FWIW, this is hard to read, and would benefit (as a separate change) in declaring a type for URLs, such
as:

URLType = Union[str, BaseURL]

Reply:
Yes I agree with Joel shouldn’t be hard to include it in this commit 👍